subscribie/blueprints/admin/templates/admin/dashboard.html

```diff
@@ -47,11 +47,9 @@ <h3 class="card-title">Checklist</h3>
     <div class="accordion col-lg-8 mx-auto mb-3" id="accordionExample">
       <div class="card">
         <div class="card-header" id="headingOne">
-          <h5 class="mb-0">
-            <button class="btn btn-link" type="button" data-toggle="collapse" data-target="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
+            <button class="btn btn-link btn-block text-left" type="button"  data-toggle="collapse" data-target="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
```

Comment:
Ah! `btn-block` :+1: good spot @Anpoli13

Reply:
By the way I really like how you do commits with clear messages in separate commits, this makes reviewing code easy, thanks